client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -640,24 +636,27 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 
 export default connect(
 	state => {
-		const site = state.jetpackConnect.jetpackConnectAuthorize &&
-				state.jetpackConnect.jetpackConnectAuthorize.queryObject &&
-				state.jetpackConnect.jetpackConnectAuthorize.queryObject.site
+		const queryObjectSite = state.jetpackConnect.jetpackConnectAuthorize &&
+			state.jetpackConnect.jetpackConnectAuthorize.queryObject &&
+			state.jetpackConnect.jetpackConnectAuthorize.queryObject.site;
+
+		const site = queryObjectSite
 			? getSiteByUrl( state, state.jetpackConnect.jetpackConnectAuthorize.queryObject.site )
```

Comment:
Hmm, that rename may not be that easy ... that queryObject is pretty much what Jetpack sends to calypso, so renaming it is out of reach (until a new jetpack release). We could of course just normalize the input and rename the attribute, but I'm not sure what's our common position on that (using the same attribute names than the API or not). I remember there was a discussion some time ago, but I don't remember getting to an agreement ... what do you think about it, @aduth  ?